src/leiningen/ring/uberwar.clj

```diff
@@ -43,3 +64,12 @@
        project war-name
        :profiles-to-merge [:uberjar]
        :additional-writes jar-entries)))
+
+(comment
```

Comment:
Comment should be removed.

Reply:
Not sure what the harm is, but ok

Reply:
are you sure you want this comment removed? It's a quick and easy way to test the fix.

Reply:
Can you explain the purpose of the comment? It's not obvious to me why it was left in.

Reply:
This project has zero tests. I have added a new function which changes the jar-name as it's being copied into the uberwar. I was using the expression inside the comment as a way to test my work during development, and left it in so that you can test/try it yourself (or anyone else that might come back to it in the future).  That is all...why would you not want it in? you've not asked me to remove any other comments, so why is this special?